.github/workflows/pythonpackage.yml

```diff
@@ -15,15 +15,23 @@ jobs:
     strategy:
       matrix:
         os: [macos-10.15, ubuntu-latest]
-        python-version: [3.6, 3.7]
+        python-version: [3.7]
 
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - name: Install dependencies Ubuntu
+      if: ${{matrix.os == 'ubuntu-latest'}}
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install -e ".[dev]"
+        pip install flake8 pytest black
+        pip install torch>=1.6.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
+    - name: Install dependencies MacOS
+      if: ${{matrix.os == 'macos-10.15'}}
       run: |
         python -m pip install --upgrade pip setuptools wheel
         pip install -e ".[dev]"
```

Comment:
OK, the Ubuntu checks were successful. Now, you can add `pip install torch` before or after this line to install it for MacOS as well (this is needed because `torch` is removed entirely from `setup.py`).

Reply:
d0h.